packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -47,6 +50,8 @@ const SquadStat = ({ count, label }: SquadStatProps) => (
   </span>
 );
 
+const MAX_PRIVILEGED_MEMBERS = 3;
```

Comment:
Could this be moved up to config.ts or constants.ts?

Reply:
I was thinking that since this is used only here, we can maybe keep it. Do you know if we would be using this somewhere else?

Reply:
Yeah, totally makes sense. Also to avoid duplication and issues in the future. Let me find a place to store this.

Reply:
https://github.com/dailydotdev/apps/blob/main/packages/shared/src/lib/config.ts or https://github.com/dailydotdev/apps/blob/main/packages/shared/src/lib/constants.ts?

Reply:
Maybe config is a better one.

Reply:
Yeah, also a bit more empty. Lots of room for new constants. Thank you!